staging/src/k8s.io/api/core/v1/well_known_taints.go

```diff
@@ -43,6 +43,6 @@ const (
 	TaintNodeNetworkUnavailable = "node.kubernetes.io/network-unavailable"
 
 	// TaintNodePIDPressure will be added when node has pid pressure
-	// and removed when node has enough disk.
+	// and removed when node has enough process room.
```

Comment:
```suggestion
	// and removed when node has enough process capacity.
```

Reply:
Thanks for reviewing.

But, because commit 51d563824d552bf88edb53a44fa669a4227eaeb3 fixed this issue, I will close this PR.